### PR TITLE
Feature/home screen actions [PIL-357,  PIL-358,  PIL-359]

### PR DESCRIPTION
**Pull request by vyvil**
This PR adds 3 action buttons to Home screen and action modals to allow users to pick wallets for those actions

### CODE REVIEW

File: src/components/ActionModal/ActionModal.js [hunk 1]
```diff
@@ -28,28 +28,36 @@ import Icon from 'components/Icon';
 import { spacing, fontStyles } from 'utils/variables';
 import type { Theme } from 'models/Theme';
 
+
```
Comment (grilix) [inline src/components/ActionModal/ActionModal.js:31]: What's the rule for these extra spaces?

Comment (vyvil) [reply src/components/ActionModal/ActionModal.js:31]: I've been following this: https://pillarwallet.slack.com/archives/CEEV48867/p1583529140199300

Comment (grilix) [reply src/components/ActionModal/ActionModal.js:31]: Nice! Totally missed that one. Thanks!

File: src/screens/Home/ActionButtons.js [hunk 2]
```diff
@@ -0,0 +1,380 @@
...
+      bitcoinAddresses,
+    } = this.props;
+    const fiatCurrency = baseFiatCurrency || defaultFiatCurrency;
+
+    const keyWallet = _accounts.find(({ type }) => type === ACCOUNT_TYPES.KEY_BASED) || {};
+    const accountsToShow = [keyWallet];
+
+    if (smartWalletFeatureEnabled) {
+      const smartWallet = _accounts.find(({ type }) => type === ACCOUNT_TYPES.SMART_WALLET) || {};
+      accountsToShow.unshift(smartWallet);
```
Comment (grilix) [inline src/screens/Home/ActionButtons.js:166]: If smartwallet doesn't exist, `{}` gets added as an account, breaking `calculateBalanceInFiat(rates, balances[id], fiatCurrency)` below.

```suggestion
      const smartWallet = _accounts.find(({ type }) => type === ACCOUNT_TYPES.SMART_WALLET);
      if (smartWallet) {
        accountsToShow.unshift(smartWallet);
      }
```

**Comment by vyvil** [comment]
Good catches, fellas! I'll fix them up today. Thanks! :)

**Pull request merged**